packages/ui-extensions-react/src/surfaces/admin/components/Link/examples/app-link.example.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import {
+  render,
+  Link,
+} from '@shopify/ui-extensions-react/admin';
+
+
+render('Playground', () => <App />);
+
+function App() {
+  return (
+    <Link href="app://bar">
```

Comment:
Thanks for adding these custom protocols - I think our preferred url pattern is to omit the `//` - Can we make all of these exclude the slashes? Ex. `app:bar`